src/invitations.ts

```diff
@@ -51,7 +52,7 @@ export default class Invitations {
 
     try {
       const response = await fetch(
-        new URL(this.invitationsEndpoint, this.invitationsUrl).toString(),
+        new URL(`${domainId}/${this.invitationsEndpoint}`, this.invitationsUrl).toString(),
```

Comment:
When it comes to the `domainId` here why not use the `domainId` present in the invitation object? https://github.com/nyagamunene/magistrala/blob/e6f9560671db51f4449af49944a94e3ebc6df2d7/pkg/sdk/go/invitations.go#L24